components/dashboard/src/app/AppBlockingFlows.tsx

```diff
@@ -53,5 +56,9 @@ export const AppBlockingFlows: FC = ({ children }) => {
         return <UserOnboarding user={user} />;
     }
 
+    if (!org.data) {
+        return <OrgNamingStep onComplete={() => {}} />;
```

Comment:
**thought**: Alternatively we could reuse the classic org creation with a twist, like hidden navigation, no-cancel button, etc or even include their user avatar to be able to still have access to their user settings.

| BEFORE | AFTER :a: | AFTER :b: |
|-|-|-|
| <img width="1512" alt="Screenshot 2023-05-22 at 13 55 47" src="https://github.com/gitpod-io/gitpod/assets/120486/f67e969c-b78d-4cda-8a96-87a55066d4b3"> | <img width="1512" alt="Screenshot 2023-05-22 at 14 10 54" src="https://github.com/gitpod-io/gitpod/assets/120486/7d3cd5b9-5877-4e7e-ab9b-8a3b660a5838"> | <img width="1440" alt="Welcome" src="https://github.com/gitpod-io/gitpod/assets/120486/255bd162-139d-47b7-b9ad-8a5bc65c3cd5"> |

Reply:
Why do you want to go with two variants of creating and naming an org?

Reply:
Oh, this was only a _thought_ in case we couldn't or didn't want to adjust the naming page above in https://github.com/gitpod-io/gitpod/pull/17697#discussion_r1200353634.

The only reason worth having two different variants is to create better [visibility of system status](https://www.nngroup.com/articles/visibility-system-status/) and [user control](https://www.nngroup.com/articles/user-control-and-freedom/) when a user does not belong to any team. If we go with the changes in the naming page above[[1](https://github.com/gitpod-io/gitpod/pull/17697#discussion_r1200353634)], there's no need to have a separate variant here.